couchexport/writers.py

```diff
@@ -347,19 +347,27 @@ def _init_table(self, table_index, table_title):
 
     def _write_row(self, sheet_index, row):
         sheet = self.tables[sheet_index]
+
         # Source: http://stackoverflow.com/questions/1707890/fast-way-to-filter-illegal-xml-unicode-chars-in-python
         dirty_chars = re.compile(
-            u'[\x00-\x08\x0b-\x1f\x7f-\x84\x86-\x9f\ud800-\udfff\ufdd0-\ufddf\ufffe-\uffff]')
+            u'[\x00-\x08\x0b-\x1f\x7f-\x84\x86-\x9f\ud800-\udfff\ufdd0-\ufddf\ufffe-\uffff]'
+        )
+
+        def get_write_value(value):
+            if isinstance(value, (int, long, float)):
+                return value
+            if isinstance(value, str):
```

Comment:
That would fail as follows:
```
>>> unicode(u'ç', encoding='utf-8')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: decoding Unicode is not supported
```
I kept the logic the same as before, just rewritten in an easier-to-read format.